src/mongo/MongoScrapbook.ts

```diff
@@ -74,7 +74,12 @@ export function getAllCommandsFromTextDocument(document: vscode.TextDocument): M
 
 async function executeCommands(activeEditor: vscode.TextEditor, database: IAzureParentNode<MongoDatabaseTreeItem>, extensionPath, editorManager: CosmosEditorManager, tree: AzureTreeDataProvider, context: IActionContext, commands: MongoCommand[]): Promise<void> {
 	for (let command of commands) {
-		await executeCommand(activeEditor, database, extensionPath, editorManager, tree, context, command);
+		try {
+			await executeCommand(activeEditor, database, extensionPath, editorManager, tree, context, command);
+		} catch (e) {
+			e.message = `${command.text.split('(')[0]}, line ${command.range.start.line + 1}, character ${command.range.start.character + 1}: ${e.message}`;
```

Comment:
The reason for using split is unclear (and very suspect).

Reply:
It arose out of `command.collection` not always being defined. I wanted the command prefix that would hold for both of : 
1. `db.collection1.insertMany`
2. `db.getCollectionInfos()`

It seems like printing the substring till the first open bracket is a simple way to extract a well-represented prefix for both cases. 

I myself don't find it cleanest or the most obvious, I'm open to suggestions.